lib/core-common/src/utils/template.ts

```diff
@@ -27,10 +25,7 @@ export function getPreviewHeadTemplate(
   configDirPath: string,
   interpolations?: Record<string, string>
 ) {
-  const base = fs.readFileSync(
-    path.resolve(__dirname, '../templates/base-preview-head.html'),
-    'utf8'
-  );
+  const base = fs.readFileSync(`${sync(__dirname)}/templates/base-preview-head.html`, 'utf8');
```

Comment:
Thinking about these templates, they really should be part of `core-server`, no?

Reply:
I have no idea what the package naming means. What is `core-common` for precisely?